lib/game.rb

```diff
@@ -0,0 +1,58 @@
+
+class Game
+
+
+  attr_reader :turn_controller, :end
+
+  attr_accessor :players
+
+  def initialize(p1 = nil, p2 = nil)
+    @players = [p1, p2]
+    @end = false
+
+    @beats = {
+      :Rock => [:Scissors, :Lizard],
```

Comment:
Great way of laying out this hash for calculating what action beats what, it is very clear and readable!